puppet/zulip/files/postgresql/process_fts_updates

```diff
@@ -43,6 +43,8 @@ sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../.
 try:
     os.environ['DJANGO_SETTINGS_MODULE'] = 'zproject.settings'
     from django.conf import settings
+    if settings.USING_PGROONGA:
+        exit()
```

Comment:
Right, so the story here is that Zulip does this deferral as a performance optimization.  Updating the full-text search index is a substantial portion of the total work involved in `do_send_message`, so rather than blocking on doing that before finishing the message sending operation, we do it in the background after the message is sent.  So this issue isn't search performance, it's message sending performance.

I think it would be good to use the same mechanism with PGroonga if there isn't a technical constraint that makes this difficult, since otherwise sites using this will experience substantially slower message sending performance.